data/systemd/snapd.manage

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+#
+# This script will start snapd from a base snap. This script knows
+# what units to add/start and remove/stop the current version of
+# snapd.
+#
+# Arguments:
+# $1: start|stop
+# $2: prefix of the installed snapd snap (e.g. /snap/snapd/42/)
+#     (on a fedora base this might be e.g. /var/lib/snaps...)
+
+set -e
+
+CMD="$1"
+PREFIX="$2"
+
+if ! grep "ID=ubuntu-core" /etc/os-release && ! grep "ID_LIKE=ubuntu-core"; then
```

Comment:
Please `grep -q`

Reply:
are we expecting a fedora bootable base to set ID_LIKE=ubuntu-core ?

Reply:
also from the discussion we had yesterday it sounded like gustavo wanted to use the unit that tries various snapd snaps also on classic, but maybe that is something for later?

Reply:
The point about ubuntu core is spot on but I think we can add a TODO comment for that.

Reply:
Isn't the 2nd grep missing a file? `... && ! grep "ID_LIKE=ubuntu-core" /etc/os-release`

Reply:
On a side note, IIRC /etc/os-release can be sourced